src/components/BookList/index.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/react-in-jsx-scope */
+
+import styles from './styles.module.scss';
+
+function BookList({ books }: any) {
+  const listBooks = books.data?.page;
+
+  return (
+    <>
+      {listBooks?.map((book: any) => (
```

Comment:
can you add interface here for book
```suggestion
      {listBooks?.map(({id, image_url, title, author }: interface) => (
```